readme.md

```diff
@@ -1,212 +1,234 @@
+
+- [ ] Update the API so that you can successfully delete a record. Use the [Model.findOneAndDelete()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndDelete) method.
+
+---
+
+- [ ] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the "_id", "title", "start_year", and "season_count" on all the above requests.
+
+* **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
+
+* **Your Answer:** 
+
+---`Model.create()` failed to return a response, even though the creation of the document was successful in the DB. I am not entirely sure that i understand the error in the terminal, but it seems as though i am dealing with a promise rejection and would need to use something like `.catch()` to handle with an error message, function, etc. Is it because ``Model.create()` is not returning a query?
+
+- [ ] Modify your `GET /api/series` route so that you can search through the content based off of query parameters. For example, if your request was `GET /api/series?start_year=1997`, it would return all series that start in 1997.
+
+---
+
+- [ ] At the moment, there is no validation set on creating any series. Add validation so that each of the three fields are required in order to post to series. Handle any errors that arise with a status code 400 and a brief, standardized message.
+
+* **Question:** You may notice that you can add custom error messages to individual fields failing. Try adding these and take a look at the error message received. How can you make use of those specific messages for each field?
+
+* **Your Answer:**
+
+--- I see the specific messaging coming back in the terminal, but i'm unclear on how to take the `message` key/value pair and parse it to my error message.
+
+- [ ] With Mongo, it is simple to create complex data structures. Add a `characters` field to your Series model that is an array of objects. Each object should have a `name` and `image_url`. Only the `name` field on the character should be required. _Note: Don't forget to change your select statements to include the `characters` field!_
+
+* **Question:** Take a look at the response from making a new series with a character. What field was created that you did not have to define?
+
+* **Your Answer:**
+
+--- So I consistently ran into the 400 error defined in my try/catch statements, but when checking Atlas I can see the document was created. My series.js file is also uploaded for reference. Why might this be? To answer the question however, the `_id` field was automatically created by Mongoose.

```

Comment:
Let me know if you're still having this issue. I would try and log out the error your receiving in the `catch` portion to narrow this down.